sdk/extensions/Azure.Extensions.AspNetCore.Configuration.Secrets/tests/AzureKeyVaultConfigurationTests.cs

```diff
@@ -437,6 +438,60 @@ public void ReplaceDoubleMinusInKeyName()
             }
         }
 
+        [Test]
+        public void HandleCollisions()
+        {
+            var client = new Mock<SecretClient>();
+            SetPages(client,
+                new[]
+                {
+                    CreateSecret("Section---Secret1", "Value1")
+                },
+                new[]
+                {
+                    CreateSecret("Section--Secret1", "Value1")
```

Comment:
Seems these should have different values to assert you actually got the right one, yes?

Reply:
You are right, I have fixed that.